script.js

```diff
@@ -1,14 +1,22 @@
+const form = document.getElementById('form');
 const container = document.getElementsByClassName('container');
 const searchBar = document.getElementsByClassName('searchbar');
```

Comment:
you can add at the end of this line of code "[0]" so you don't have to write [0] frequently below. 

const searchBar = document.getElementsByClassName('searchbar')**[0]**;